docs/templates/index.md

```diff
@@ -0,0 +1,12 @@
+---
+---
+
+When writing docs, start with one of the five templates we provide:
+
+Template | When to Use This | Download Link
+--- | --- | --- | ---
+[Overview](/docs/templates/overview/) | When you want to write an overview of X and its most commonly used elements (e.g. overview of APIs) | [download this](https://github.com/kubernetes/kubernetes.github.io/tree/master/docs/templates/overview.md)
+[Concept or in-depth guide](/docs/templates/concept-guide/) | When you want to explain X and how it works in detail (e.g. Deployment object concept) | [download this](https://github.com/kubernetes/kubernetes.github.io/tree/master/docs/templates/concept-guide.md)
+[Quickstart](/docs/templates/quickstart/) | When you want to get users working with the technology quickly by completing an introductory task within 5 minutes | [download this](https://github.com/kubernetes/kubernetes.github.io/tree/master/docs/templates/quickstart.md)
```

Comment:
I think we should eliminate "Quickstart". I don't understand the difference between it and the other types of docs.